tests/mock_server.py

```diff
@@ -29,7 +29,7 @@ def start_service(service_name, host, port):
             requests.get(url, timeout=0.5, proxies=_proxy_bypass)
             break
         except requests.exceptions.ConnectionError:
-            time.sleep(0.5)
```

Comment:
instead of bumping this to 1.5s, could you instead bump the range at the top?  That way if it takes < 1.5s to start the service the tests will run faster

Reply:
Yeah good spot, that's a much better idea.

Reply:
👍